src/cc2olx/olx.py

```diff
@@ -163,7 +163,10 @@ def _add_olx_nodes(self, element, course_data, tags):
 
             for child in children:
                 if "title" in element_data:
-                    child.setAttribute("display_name", element_data["title"])
+                    if "%" in element_data["title"]:
```

Comment:
We could be more robust with this, and ensure that only valid characters are in the title, not just check for the percentage

Reply:
Good point! I'll spend a little bit of time updating this.